### PR TITLE
Fix upload plugin

### PR DESCRIPTION
**Pull request by jorgeRambla**
### What does it do?

Remove count controller from upload plugin

### Why is it needed?

Detected that count controller (& route) is defined, but count service is not defined, after raising this in an issue, staff mark as TODO to remove the count controller for upload.

### How to test it?

Plugin upload should not contain neither the count route, neither the controller method/

### Related issue(s)/PR(s)

fixes https://github.com/strapi/strapi/issues/13011

### CODE REVIEW

**Comment by jorgeRambla** [comment]
Waiting for work flow approval

**Review by Convly — APPROVED** [review]
LGTM, thanks :slightly_smiling_face:

**Pull request merged**
